beastify/popup/choose_beast.js

```diff
@@ -26,19 +26,24 @@ Then get the active tab and send "beastify.js" a message
 containing the URL to the chosen beast's image.
 */
 document.addEventListener("click", function(e) {
-  if (!e.target.classList.contains("beast")) {
+  if (e.target.classList.contains("beast")) {
+    var chosenBeast = e.target.textContent;
+    var chosenBeastURL = beastNameToURL(chosenBeast);
+
+    chrome.tabs.executeScript(null, {
+      file: "/content_scripts/beastify.js"
+    });
+
+    chrome.tabs.query({active: true, currentWindow: true}, function(tabs) {
+      chrome.tabs.sendMessage(tabs[0].id, {beastURL: chosenBeastURL});
+    });
+    
     return;
```

Comment:
I'm keeping it as good practice. If we can't use a switch case to handle this (which would use breaks), we should at least have the return statements at the ifs as some sort of equivalent.